.github/workflows/deploy-prod.yaml

```
@@ -0,0 +1,65 @@
+name: Deploy to production
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-test-and-deploy-production:
+    name: Build and deploy to production environment on merge to master branch
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup gcloud CLI
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          service_account_email: ${{ secrets.GCP_SA_DIGDIR_FDK_GCR_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_DIGDIR_FDK_GCR_KEY }}
+
+      - name: Configure docker to use the gcloud command-line tool as a credential helper
+        run: gcloud auth configure-docker
+
+      - name: Build Docker image
+        run: docker build . -t eu.gcr.io/digdir-fdk-infra/fdk-cms-service:$GITHUB_SHA -t eu.gcr.io/digdir-fdk-infra/fdk-cms-service:latest --build-arg GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
+
+      - name: Publish docker image to Google Cloud
+        run: docker -- push eu.gcr.io/digdir-fdk-infra/fdk-cms-service -a
+
+      - name: Apply Helm template
+        run: |
+          helm repo add fdk https://informasjonsforvaltning.github.io/helm-chart/
+          helm fetch --untar --untardir ./helm fdk/fdk-cms-service
+          helm template --set DOCKER_IMAGE_NAME="eu.gcr.io/digdir-fdk-infra/fdk-cms-service:$GITHUB_SHA" -f ./helm/fdk-cms-service/values/prod.yaml ./helm/fdk-cms-service/ > ./kubectlapply.yaml
+          chmod o+w ./kubectlapply.yaml
+          cat kubectlapply.yaml
+
+      - name: Get kubeconfig file from GKE
+        uses: machine-learning-apps/gke-kubeconfig@master
+        with:
+          application_credentials: ${{ secrets.DIGDIR_FDK_PROD_AUTODEPLOY }}
+          project_id: digdir-fdk-prod
+          location_zone: europe-north1-a
+          cluster_name: digdir-fdk-prod
+
+      - name: Deploy to production
+        uses: docker://bitnami/kubectl:latest
+        env:
+          KUBECONFIG: '/github/workspace/.kube/config'
+        with:
+          args: "apply -f ./kubectlapply.yaml --force"
```

`kubectl` kommer ferdig installert, men om man trenger en spesifik versjon av den, så kan dette være et alternativ:

```suggestion
      - uses: azure/setup-kubectl@v1
      - run: kubectl apply -f ./kubectlapply.yaml --force
        env:
          KUBECONFIG: '/github/workspace/.kube/config'
```

Hvis, ikke kan vi egentlig bare gjøre dette:

```suggestion  
      - run: kubectl apply -f ./kubectlapply.yaml --force
        env:
          KUBECONFIG: '/github/workspace/.kube/config'
```

Lurer på om KUBECONFIG henter sin konfigurasjon som standard fra `~/.kube/config`, så den kan vel kanskje fjernes?